flagset.go

```diff
@@ -55,12 +55,39 @@ func (f *FlagSetFiller) Fill(flagSet *flag.FlagSet, from interface{}) error {
 	}
 }
 
+var supportedStructList = make(map[string]struct{})
```

Comment:
Can you add a comment here about the need to give awareness of structs that don't need to be walked into?